src/main/java/org/geonetwork/index/model/record/IndexRecord.java

```diff
@@ -647,7 +650,12 @@ public Map<String, Object> getOtherProperties() {
   @JsonAnySetter
   public void handleUnrecognizedField(String name, Object value) {
     try {
-      IndexRecord.class.getDeclaredField(name);
+      Field unexpected = IndexRecord.class.getDeclaredField(name);
+      if (unexpected != null) {
+        System.out.printf(
+          "Handle unrecognized filed %s with value %s for record %s unrecognized conflicts with field %s",
+          name, value, this.getMetadataIdentifier(), unexpected.getName());
+      }
```

Comment:
I understand that this was intended to be executed for a side effect, but am addressing a warning. We may as well cover the case if the field is defined.